frameworks/kafka/src/main/resources/kafka_svc.yml

```diff
@@ -0,0 +1 @@
+../dist/kafka_svc.yml
```

Comment:
OK I'll rename it back in #440. Would be good to get this merged and no reason to bump the CI for this.